src/os/bluestore/BlueStore.cc

```diff
@@ -5747,25 +5747,24 @@ void BlueStore::_do_write_small(
   }
 
   // new blob.
-  int64_t blob;
-  b = o->onode.add_blob(&blob);
-  b->length = min_alloc_size;
+  b = o->blob_map.new_blob(c->cache);
+  b->blob.length = min_alloc_size;
```

Comment:
Yeah we could.. I'm inclined to keep the BlobMap functions only concerned with the map structure though and not the blob data itself.